wikipron/scrape.py

```diff
@@ -41,8 +42,9 @@ def _scrape_once(data, config: Config) -> Iterator[WordPronPair]:
         ):
             continue
         request = session.get(_PAGE_TEMPLATE.format(word=word), timeout=10)
+        word = unicodedata.normalize("NFD", word)
         for word, pron in config.extract_word_pron(word, request, config):
-            yield word, pron
+            yield word, unicodedata.normalize('NFC', pron)
```

Comment:
This looks good to me but use double-quotes here.

Reply:
Also, could you add a line of documentation here:

# Pronunciation processing is done in NFD-space; we convert back to NFC afterwards.